server/websocket/src/connection.rs

```diff
@@ -313,7 +313,7 @@ where
                     };
 
                     // Construct a subscription
-                    let target = QueryTarget::Deployment(deployment.clone());
+                    let target = QueryTarget::Deployment(deployment.clone(), Default::default()); // TODO: resolve and validate a version number from WebSocket connection
```

Comment:
- [ ] resolve version of WebSocket connections

Reply:
I think we can leave this as-is since there are no subscriptions on the network anyway